### PR TITLE
command to migrate dash eggs

### PR DESCRIPTION
**Pull request by krotkiewicz**

### CODE REVIEW

File: shub/migrate_eggs.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    def start(self):
+        if self.eggs:
+            self.migrate_eggs()
+
+        self.migrate_requirements_txt()
+
+        self.conf.save(self.sh_yml)
+
+    def migrate_eggs(self):
+        eggsdir = click.prompt('Provide directory for eggs', default='./eggs')
```
Comment (dangra) [inline shub/migrate_eggs.py]: prompting for eggsdir is too much, let's just show where eggs are stored (per egg) if any.

File: shub/migrate_eggs.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        try:
+            os.mkdir(eggsdir)
```
Comment (dangra) [inline shub/migrate_eggs.py]: no point on creating the directory if there aren't eggs

File: shub/migrate_eggs.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        except OSError:
```
Comment (dangra) [inline shub/migrate_eggs.py]: I guess the catch is for "File already exists", so the usual flow is:

```python
try:
    os.mkdir(PATH)
except OSError as exc:
    if exc.errno != errno.EEXIST:
        raise
```

**Comment by dangra** [comment]
Btw, it lacks tests.

File: shub/migrate_eggs.py (hunk 1 above)
Comment (krotkiewicz) [inline shub/migrate_eggs.py]: fixed

File: shub/migrate_eggs.py (hunk 2 above)
Comment (krotkiewicz) [inline shub/migrate_eggs.py]: not creating, the if is in start method

File: shub/migrate_eggs.py (hunk 3 above)
Comment (krotkiewicz) [inline shub/migrate_eggs.py]: ok

**Comment by krotkiewicz** [comment]
Tests added

**Comment by dangra** [comment]
Hi @krotkiewicz, the changes look good but travis-ci and appveyor still thinks the testsuite is broken.

**Comment by krotkiewicz** [comment]
@dangra probably due to some changes to pyinstaller or setuptools.

I managed to fix it in https://github.com/scrapinghub/shub/pull/209
I will rebase on top of it after you merge it.

**Pull request merged**